tests/unit/conftest.py

```diff
@@ -26,3 +28,55 @@ def naur_guild(mocker: MockerFixture, create_role):
             create_role(enums.Role.EXILED),
         ]
     )
+
+
+@pytest.fixture
+def create_db_user(mocker: MockerFixture):
+    def __create_db_user(
+        user_id: Optional[int] = None,
+        discord_user_id: Optional[str] = None,
+        discord_guild_id: Optional[str] = None,
+        is_mod: Optional[bool] = None,
+        temporary_points: Optional[int] = None,
+        permanent_points: Optional[int] = None,
+        last_infraction_timestamp: Optional[datetime] = None,
+        get_strike_points: Optional[int] = None,
+    ):
+        mocked_user = mocker.Mock(spec=User)
+        mocked_user.user_id = user_id
+        mocked_user.discord_user_id = discord_user_id
+        mocked_user.discord_guild_id = discord_guild_id
+        mocked_user.is_mod = is_mod
+        mocked_user.temporary_points = temporary_points
+        mocked_user.permanent_points = permanent_points
+        mocked_user.last_infraction_timestamp = last_infraction_timestamp
+        mocked_user.get_strike_points = mocker.Mock(return_value=get_strike_points)
+        return mocked_user
```

Comment:
This all can be done in the `mocker.Mock` call, the only attribute I know where that won't work is `name`